### PR TITLE
Adds audio querying to MultimodalQ&A Example

### PR DESCRIPTION
**Pull request by mhbuehler**
## Description

Adds the option to speak into the browser microphone or upload an audio file to the main query tab. The speech audio is POSTed as a base64 string to the MultimodalQnA gateway where it is transcribed to text by the ASR whisper service and then passed on to the LVM service. 

Note: This PR has been updated so that the changes to the megaservice are included in this repo in `multimodalqna.py`, so a corresponding PR to GenAIComps is not needed.  

## Issues

This is a part of the [MultimodalQnA Image & Audio Enhancements RFC](https://github.com/opea-project/docs/blob/main/community/rfcs/24-10-02-GenAIExamples-001-Image_and_Audio_Support_in_MultimodalQnA.md)

## Type of change

- [x] New feature (non-breaking change which adds new functionality)

## Dependencies

N/A

## Tests

New tests were added to the example's `README.md`, `test_compose_on_xeon.sh`, and `test_compose_on_gaudi.sh`.

### CODE REVIEW

File: MultimodalQnA/tests/test_compose_on_xeon.sh [hunk 1]
```diff
@@ -20,8 +20,9 @@ export caption_fn="apple.txt"
 
 function build_docker_images() {
     cd $WORKPATH/docker_image_build
-    git clone https://github.com/opea-project/GenAIComps.git && cd GenAIComps && git checkout "${opea_branch:-"main"}" && cd ../
-
+    # TODO: Revert to the clone of opea-project "main" before merge
+    # git clone https://github.com/opea-project/GenAIComps.git && cd GenAIComps && git checkout "${opea_branch:-"main"}" && cd ../
+    git clone --single-branch --branch="mmqna-audio-query" https://github.com/mhbuehler/GenAIComps.git
```
Comment (chensuyue) [inline MultimodalQnA/tests/test_compose_on_xeon.sh:25]: Please revert this back before PR merge.

File: MultimodalQnA/tests/test_compose_on_xeon.sh [hunk 2]
```diff
@@ -20,8 +20,9 @@ export caption_fn="apple.txt"
...
     echo "Build all the images with --no-cache, check docker_image_build.log for details..."
     service_list="multimodalqna multimodalqna-ui embedding-multimodal-bridgetower embedding-multimodal retriever-multimodal-redis lvm-llava lvm-llava-svc dataprep-multimodal-redis"
```
Comment (chensuyue) [inline MultimodalQnA/tests/test_compose_on_xeon.sh:27]: New images should be included in this service list, or it won't be build. If all the images in `build.yaml` required in this test, just remove the service_list, which means build all the images.

Comment (mhbuehler) [reply MultimodalQnA/tests/test_compose_on_xeon.sh:27]: Understood, thank you!

**Review by ashahba — APPROVED** [review]
LGTM but don't merge please unless the branch is changed to point to `main` again.

**Review by ashahba — CHANGES_REQUESTED** [review]
Holding off on this PR until https://github.com/opea-project/GenAIComps/pull/974 is merged.

File: MultimodalQnA/tests/test_compose_on_xeon.sh (hunk 1 above)
Comment (mhbuehler) [reply MultimodalQnA/tests/test_compose_on_xeon.sh]: This is done

**Review by ashahba — APPROVED** [review]
LGTM!

**Review by lvliang-intel — APPROVED** [review]

**Pull request merged**